annotations/oarhole.md

```diff
@@ -0,0 +1,31 @@
+**_oarhole, oarport_** (English); _årehul_ (Danish); _Riemenloch_ (German)
+
+_**hár** f., pl. háir_ (Old Norse) [citation: [prose](https://onp.ku.dk/onp/onp.php?o31447)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=31982)]     
+_**hamla** f., pl. hǫmlur_ (Old Norse) [citation: [prose](https://onp.ku.dk/onp/onp.php?o31115)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=31626)]   
+
+
+  Holes through an upper strake of the hull that acted as fulcrums for rowing the oars of a Viking ship.
+
+<div align="center">
+  
+  ![oarholes from the Gokstad ship, exterior](../images/Oarholes_Gokstad_Ex.jpg)  
+  Reconstruction drawing of a section of the upper strakes and oarports of the Gokstad ship from the exterior (Nicolaysen Pl. I, Fig. 1).
+
+  ![oarholes from the Gokstad ship, interior](../images/Oarholes_Gokstad_In.jpg)  
+  Interior of the same portion of the Gokstad ship, showing each oarhole centered in a "room" (Nicolaysen Pl. I, Fig. 4).
+
+</div>
+
+Ships needed high sides to keep out waves at sea, so oars were put through holes in an upper strake rather than rowlocks atop the gunwale used in smaller boats (Crumlin-Pedersen 128). The number of oarholes, and therefore oarsmen, determined the size of a ship's company, and taxes were often levied based on the amount of oarports present on a ship (Jesch 156).
```

Comment:
```suggestion
Ships needed high sides to keep out waves at sea, so oars were put through holes in an upper strake rather than rowlocks atop the gunwale as in smaller boats (Crumlin-Pedersen 128). The number of oarholes, centered on "rooms" between the beams, determined the size of a ship's company, and taxes were often levied based on the amount of oarports present on a ship (Jesch 156).
```